app/assets/javascripts/fae/form/_validator.js

```diff
@@ -51,6 +51,13 @@ Fae.form.validator = {
       if ($(".field_with_errors").length) {
         $('.alert').slideDown('fast').removeClass('hide').delay(3000).slideUp('fast');
       }
+
+      if ($('.invalid').length) {
```

Comment:
Also, the Travis build is failing, you can click to view the details to see which tests are failing. At first glance it looks like it's around nested forms.